tests/behat/features/asp/homepage/asp-social.feature

```diff
@@ -0,0 +1,34 @@
+@asp
+@api
+@javascript
+
+Feature: verify the social links on the homepage
+
+  Scenario:verify the social links on the homepage
+    Given I am not logged in
+    When I am on the homepage
+    Then I should see an "a[href='https://twitter.com/ACCESSforCI']" element
+    # clicking giving bad responses - so relying on existence of element in prev step
+    #When I click the "a[href='https://twitter.com/ACCESSforCI']" element
+    #And I wait 2 seconds
+    #Then print current URL
+    #Then I should get a "200" HTTP response
+
+    When I am on the homepage
+    Then I should see an "a[href='https://www.youtube.com/c/ACCESSforCI']" element
+    # clicking giving bad responses - so relying on existence of element in prev step
+    #When I click the "a[href='https://www.youtube.com/c/ACCESSforCI']" element
+    #And I wait 2 seconds
+    #Then I should get a "200" HTTP response
+
+    When I am on the homepage
+    When I click the "a[href='https://www.facebook.com/ACCESSforCI']" element
+    And I wait 2 seconds
+    Then I should get a "200" HTTP response
+
```

Comment:
Seems like we can remove the 3 scenarios above this. Checking that the <a> elements exist with the proper links is sufficient. We don't need to verify the 200 response here.

Reply:
Ok, will update

Reply:
updated.